apps/air-discount-scheme/backend/src/app/modules/flight/test/integration/flight.service.spec.ts

```diff
@@ -25,23 +25,23 @@ describe('create', () => {
         originalPrice: 50000,
         discountPrice: 30000,
         date: new Date('2021-03-12T12:35:50.971Z'),
-        cooperation: null,
+        cooperation: '',
```

Comment:
null is actually the expected behaviour. The column allows null with no default value.

Reply:
Checkout the latest commit, I made the cooperation attribute optional (as it should have been) and I removed cooperation from the tests where they were null. That should be more accurate to when a request comes in and has no cooperation.